From 19c756b65b07bff23122bc90d3e651791d6f0b19 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sun, 20 Oct 2024 22:44:37 +0200
Subject: [PATCH] Remove image rotation feature

It doesn't work well, because it leaves temporary image files on /tmp after
closing the app, and the upstream developer has no intention of fixing this.

More details:
https://gitlab.gnome.org/World/Letterpress/-/merge_requests/21
---
 src/window.py | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/window.py b/src/window.py
index 5039be9..b24e729 100644
--- a/src/window.py
+++ b/src/window.py
@@ -113,11 +113,7 @@ class LetterpressWindow(Adw.ApplicationWindow):
                     __wrong_image_type()
                     return
 
-                self.filepath = NamedTemporaryFile(suffix=f".{img_format}").name
-
-                shrunken_img = ImageOps.cover(img, (500, 500))
-                exif_rotated_img = ImageOps.exif_transpose(shrunken_img)
-                exif_rotated_img.save(self.filepath, format=img_format)
+                self.filepath = filepath
 
                 self.__convert_image(self.filepath)
         except IOError:
-- 
2.47.0

